src/content/ui-api/kubectl-gs/get-organizations.md

```diff
@@ -0,0 +1,96 @@
+---
+linkTitle: get organizations
+title: "'kubectl gs get organizations' command reference"
+description: Reference documentation on how to list organizations and get details for a single organization using 'kubectl gs'.
+weight: 50
+menu:
+  main:
+    parent: uiapi-kubectlgs
+aliases:
+  - /reference/kubectl-gs/get-organizations/
```

Comment:
aliases are only needed if the page has been moved, to have redirects from old URLs to the new URL. Not needed here.

Reply:
Removed the alias